### ISSUE TITLE
Implement new exportRow() in CustomizableTabularExporterUtilities.java

### ISSUE THREAD

**Issue by Edvinnordling** [DD2480-group-30/OpenRefine#9]
Implement a new exportRows() that that takes a `Map<String, String>` as an argument instead of `Properties`. 
Header:
`static public void exportRows(
            final Project project,
            final Engine engine,
            Properties params,
            final TabularSerializer serializer)` 

Extract code to avoid code duplications between the new and old exportRows() method

Location:
```
main/src/com/google/refine/exporters/CustomizableTabularExporterUtilities.java
  77, 24:   static public void exportRows(
```

**Comment by Edvinnordling** [comment]
We are not sure if the old exportRows() should be deprecated or not

**Comment by Edvinnordling** [comment]
Closed with PR #7
